### PR TITLE
Extend AppBuilder api with `add_system_set` and similar methods

### PR DESCRIPTION
**Pull request by TheRawMeatball**

### CODE REVIEW

**Review by Ratysz — APPROVED** [review]

**Review by Ratysz — APPROVED** [review]

File: crates/bevy_ecs/src/schedule/mod.rs [hunk 1]
```diff
@@ -153,6 +153,23 @@ impl Schedule {
         self
     }
 
+    pub fn add_system_set_to_stage(
+        &mut self,
+        stage_label: impl StageLabel,
+        system_set: SystemSet,
+    ) -> &mut Self {
+        let stage = self
```
Comment (cart) [inline crates/bevy_ecs/src/schedule/mod.rs:161]: we can cut down on the code size here like this:

```rust
self.stage(stage_label, |stage: &mut SystemStage|
    stage.add_system_set(system_set)
)
```

**Comment by cart** [comment]
Welp thats a shame. Thanks for testing @alice-i-cecile.

**Comment by cart** [comment]
Bawhaha mine failed too, but for a different reason: https://github.com/bevyengine/bevy/actions/runs/582437880

**Pull request merged**